nknd.go

```diff
@@ -209,6 +209,15 @@ func nknMain(c *cli.Context) error {
 		if err != nil {
 			return err
 		}
+
+		go func() {
+			for {
+				time.Sleep(time.Minute)
+				if node.GetConnectionCnt() == 0 {
```

Comment:
This condition can't cover those cases which neighbor list length < 3.

Reply:
We shouldn't let the network exit if # of nodes < 4 because people may want to set up a small local cluster. And the constant 3 as min number neighbors in consensus should be changed in the future anyway.